main.py

```diff
@@ -108,6 +108,28 @@ async def get_experiment_code(file: str) -> str:
     return code
 
 
+@app.post("/experiment/delete/")
+async def delete_experiment(rid):
+    """Kills the run with the specified RID.
+
+    Args:
+        rid: rid value of the target experiment.
+    """
+    remote = get_client("master_schedule")
+    remote.delete(rid)
+
+
+@app.post("/experiment/request_termination/")
```

Comment:
How about "/experiment/terminate/"?
It looks too long.

Reply:
Umm.. Why do you need this? I think `remote.delete(rid)` is enough..

Is there any reason?

Reply:
The original `scheduler` supports these two different functions, and I guess there is a difference like one between Alt+F4 and quitting by clicking the X button.

Reply:
What is difference betwwen `delete()` and `request_termination()`?

Reply:
I do not know in detail, but I made a _replica_ so I thought I need to make them for future use.

Reply:
@kangz12345 How about you?

Reply:
See also: [source code](https://m-labs.hk/artiq/manual/_modules/artiq/master/scheduler.html#Scheduler.delete)

`request_termination()` will do some proper works before deleting an experiment.
I think both are required (`delete()` could be like force-quit).